documentation/library_guide/kernel_templates/esimd/radix_sort.rst

```diff
@@ -0,0 +1,354 @@
+Radix Sort
+##########
+
+-----------------------------------------------------------
+``radix_sort`` and ``radix_sort_by_key`` Function Templates
+-----------------------------------------------------------
+
+The ``radix_sort`` and ``radix_sort_by_key`` functions sort data using the radix sort algorithm.
+The sorting is stable, ensuring the preservation of the relative order of elements with equal keys.
+The functions implement Onesweep* [#fnote1]_ algorithm variant.
+
+A synopsis of the ``radix_sort`` and ``radix_sort_by_key`` functions is provided below:
+
+.. code:: cpp
+
+   // defined in <oneapi/dpl/experimental/kernel_templates>
+
+   namespace oneapi::dpl::experimental::kt::esimd {
+
+   template <bool IsAscending = true, std::uint8_t RadixBits = 8, typename KernelParam, typename Range>
```

Comment:
```suggestion
   template <bool IsAscending = true, std::uint8_t RadixBits = 8,
             typename KernelParam, typename Range>
```
to avoid horizontal scrolling in the rendered code block.

Reply:
Done